src/eslint-config-adeira/ourRules.js

```diff
@@ -429,7 +429,7 @@ module.exports = ({
   'react/prefer-read-only-props': OFF,
   'react/prefer-stateless-function': OFF,
   'react/prop-types': OFF, // we use Flow instead,
-  'react/react-in-jsx-scope': WARN,
+  'react/react-in-jsx-scope': OFF,
```

Comment:
Possibly we should leave this and just update the rule directly in `.eslintrc.js`, and change it here with the next major version

Reply:
File `src/eslint-config-adeira/getCommonConfig.js` has `detectReactVersion` config. Would that be helpful?

Reply:
But using the new `JSX` transform is still optional, so I don't think so.

Reply:
We are just flipping it from `WARN` -> `OFF`, so I think this approach is fine.